app/models/letter_template.rb

```diff
@@ -34,6 +35,8 @@ def render(values, letter, field)
   end
 
   def telephone_number
+    return COMPLAINT_ACKNOWLEDGEMENT_LETTER_TEL_NUM if abbreviation == "complaint-acknowledgement"
```

Comment:
Not sure if its the best way to implement, however I didnt think it was a good idea to create a new template_type enum